### PR TITLE
M-T Branch BB-11

### PR DESCRIPTION
**Pull request by BivolMaksym**
Final verison of BB-11 code.

### CODE REVIEW

File: CAR M-T/Arduino code/BB11/BB11.ino [hunk 1]
```diff
@@ -0,0 +1,493 @@
+
```
Comment (2560p) [inline CAR M-T/Arduino code/BB11/BB11.ino]: The file starts with the blank line.

Please, fixup!

File: CAR M-T/Arduino code/BB11/BB11.ino [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+  startTime = millis();
+  while (millis() - startTime < driveDuration) {
+    goForward();
+  }
+  // Drive forward for a short time
+
+  startTime = millis();
+  while (millis() - startTime < turn2Duration) {
+    turnRightObstacle();
+ //Turn right for short amount of time
```
Comment (2560p) [inline CAR M-T/Arduino code/BB11/BB11.ino]: Add a space in between the forward slash and the word

File: CAR M-T/Arduino code/BB11/BB11.ino [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+
+int inter = 2;
+
+// Define the motor speed
+const int motorSpeed = 255;
+
+// Define the threshold for detecting the black line
+const int threshold = 800;
+
+//some functions declaration
```
Comment (2560p) [inline CAR M-T/Arduino code/BB11/BB11.ino]: Spacing

File: CAR M-T/Arduino code/BB11/BB11.ino [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+const int rightMotorPin2 = 5;
+const int leftMotorPin1 = 10;
+const int leftMotorPin2 = 9;
+
+// Define the servo and gripper pins
+Servo gripper;
+int gripperPin = 12;
+int pinServo = 12;
+
+//Set the int needed
```
Comment (2560p) [inline CAR M-T/Arduino code/BB11/BB11.ino]: Spacing

**Review by 2560p — CHANGES_REQUESTED** [review]

**Comment by 2560p** [comment]
@BivolMaksym, @timothyadewale, please finalise the code!

**Review by ZsomborHajzer — APPROVED** [review]
Looks good after Pauls review changes

**Review by 2560p — APPROVED** [review]
LGTM!

The changes were implemented, the typos were fixed.

**Pull request merged**